AndroidSDKCore/src/main/java/com/leanplum/internal/VarCache.java

```diff
@@ -322,6 +323,14 @@ public static boolean hasReceivedDiffs() {
     return hasReceivedDiffs;
   }
 
+  public static Map<String, Object> getVariantDebugInfo() {
+    return variantDebugInfo;
+  }
+
+  public static void setVariantDebugInfo(Map<String, Object> _variantDebugInfo) {
```

Comment:
No need for `_` in front of var.

Simply do `this.variantDebugInfo= _variantDebugInfo;`

Reply:
I used the code style that was present in the file. I think we can do a separate code style pass for the mobile code, but do not want to make this line different than the rest of the file.

Reply:
I don't see other code in this file using this pattern?

Orthodox Java approach is to use `this` to disambiguate variables when having an arg and member variable share a name.

Reply:
Just tried it - can't use "this" cuz we're using static variables. I wanna move this into an instance with singleton, but that's out of scope, so we'll just use the underscore for now

Reply:
How about Leanplum.variantDebugInfo = variantDebugInfo;